pytest_selenium/drivers/browserstack.py

```diff
@@ -14,13 +14,17 @@ class BrowserStack(Provider):
 
     @property
     def auth(self):
-        return (self.username, self.key)
+        return self.username, self.key
 
     @property
     def executor(self):
         return 'http://{0}:{1}@hub.browserstack.com:80/wd/hub'.format(
             self.username, self.key)
 
+    @property
+    def name(self):
+        return type(self).__name__
```

Comment:
We can just inherit this from `Provider`

Reply:
Yeah, that's the way it was, but then the tests failed. But if we do your suggestion below and the `uses_driver()` function, we can restore it. 👍